src/data_source/mod.rs

```diff
@@ -170,6 +177,43 @@ impl<'env> Connection<'env> {
     }
 }
 
+impl<'env> Connection<'env, AutocommitOff> {
+    /// Set autocommit mode off, triggers implicit rollback of any running transaction
```

Comment:
I guess this is a typo, i.e. you meant `Set autocommit mode on` instead of `off`?

Reply:
Thanks, I'll fix this